content/applications/services/helpdesk/overview/ratings.rst

```diff
@@ -1,51 +1,110 @@
-=======
-Ratings
-=======
+================
+Customer ratings
+================
 
-Allow customers to rate their experience with your helpdesk teams to strengthen your credibility and
-gain their trust. Reviews can also influence a customer’s decision and open space for feedback that
-can help you improve the quality of your services.
+Asking customers to rate the support they received from a *Helpdesk Team* provides an opportunity to
+gauge team performance and track customer satisfaction. Ratings can be published to the portal,
+providing customers with a general overview of the team's performance.
 
-Set up
-======
+Enable customer ratings on a helpdesk team
+==========================================
 
-Go to :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams` and enable *Ratings on
-tickets*. The feature automatically adds a default email template on the non-folded *closing
-stage(s)* of that team.
+To enable :guilabel:`Customer Ratings` on a helpdesk team, go to :menuselection:`Helpdesk -->
+Configuration --> Teams`. Select a team from the list and navigate to the settings page. Scroll to
+the :guilabel:`Performance` section, and check the box for :guilabel:`Customer Ratings`.
 
-.. image:: ratings/ratings_on_tickets.png
+.. image:: ratings/ratings-enable.png
    :align: center
    :alt: Overview of the settings page of a helpdesk team emphasizing the rating on ticket feature
-         in Odoo Helpdesk
+         in Odoo Helpdesk.
 
-To edit the email template and the stage(s) set as the closing ones, go to the Kanban view of your
-helpdesk team and click on *Settings*, then on *Edit Stage*.
+Set a ratings request email template on a stage
+===============================================
 
-.. image:: ratings/edit_stage.png
+To automatically request ratings from customers once their tickets have closed, an email template
+should be added to the appropriate stage.
+
+Once :guilabel:`Customer Ratings` has been enabled on the team's settings page, (see above) click
+the :guilabel:`Set an Email Template on Stages` link. Select a stage from the list, or click
+:guilabel:`New` to create a new stage.
+
+.. important::
+   Customers should only be asked to rate tickets once an issue has been resolved and their ticket
+   is closed. Therefore, a *ratings request* email should only be added to a stage that is
+   :guilabel:`Folded in Kanban`, as tickets in a *folded stage* are considered closed.
```

Comment:
No guilabel here.
Instead would bolden it to bring attention and then emphasize again later in the sentence.
```suggestion
   **folded** in the kanban, as tickets in a *folded stage* are considered closed.
```